README.md

```diff
@@ -0,0 +1,101 @@
+# Emybot
+
+  A cute discord bot for all your nekomimi needs.
+
+## Getting started
+
+### Prerequisites
+
+  - node.js
+
+  - discord.js: `npm install discord.js`
```

Comment:
you only need node, and a discord account. Git is optional and the other stuff will be handled with a ``npm i`` in the project root